### PR TITLE
Add kubectl-strace plugin to krew index

### PR DESCRIPTION
**Pull request by MichaelWasher**
Add kubectl-strace plugin to krew index
kubectl-strace is a plugin for capturing strace data of running Pods

### CODE REVIEW

File: plugins/strace.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: strace
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/michaelwasher/kstrace
+  shortDescription: "Captures strace logs from a running workload in the Kubernetes cluster"
```
Comment (ahmetb) [inline plugins/strace.yaml:8]: max 50 characters here (you can trim some redundant wording)

File: plugins/strace.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  description: |
+    kubectl-strace is a kubectl plugin that provides the ability to easily perform debugging of system-calls 
+    for applications running in the cluster. kubectl-strace starts a priviledged Pod and attaches an strace instance
+    to one or many running Pods, displaying or collecting the results for later review.
+    Read more documentation see: https://github.com/michaelwasher/kstrace
+  caveats: |
+    This plugin requires the RBAC authority to start privileged Pods and create Namespaces
```
Comment (ahmetb) [inline plugins/strace.yaml:15]: trailing period would be good.

File: plugins/strace.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: strace
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/michaelwasher/kstrace
+  shortDescription: "Captures strace logs from a running workload in the Kubernetes cluster"
+  description: |
+    kubectl-strace is a kubectl plugin that provides the ability to easily perform debugging of system-calls
```
Comment (ahmetb) [inline plugins/strace.yaml:10]: wrapping this paragraph by 80 characters would look great.

File: plugins/strace.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: strace
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/michaelwasher/kstrace
```
Comment (ahmetb) [inline plugins/strace.yaml:7]: I am noticing you did not import client-go auth plugins?
https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#:~:text=in%20your%20plugins.-,Import%20authentication%20plugins%20(Go),-By%20default%2C%20plugins

**Review by ahmetb — CHANGES_REQUESTED** [review]

File: plugins/strace.yaml (hunk 4 above)
Comment (MichaelWasher) [reply plugins/strace.yaml:7]: Hi @ahmetb,

Thank you for letting me know. Sorry I completely missed it. I'll fix that up.
Also, regarding the use of genericclioptions, is it an expectation to include the Kuebrentes default options(i.e. `-kubeconfig`, `--context`) or can this be excluded how I've done it here: https://github.com/MichaelWasher/kstrace/blob/master/cmd/cmd.go#L119-L122

File: plugins/strace.yaml (hunk 3 above)
Comment (MichaelWasher) [reply plugins/strace.yaml]: Sorry, you must have to say this all the time. I have done this now but can you please confirm... Is it 80 characters including or excluding the starting spaces?

File: plugins/strace.yaml (hunk 4 above)
Comment (MichaelWasher) [reply plugins/strace.yaml:7]: This import has been added, the release re-made and the SHA's updated.

File: plugins/strace.yaml (hunk 3 above)
Comment (ahmetb) [reply plugins/strace.yaml]: excl. the spaces (how it renders in user's terminal) but it's alright :)

File: plugins/strace.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  shortDescription: "Capture strace logs from a running workload"
+  description: |
+    kubectl-strace is a kubectl plugin that provides the ability to easily perform
+    debugging of system-calls for applications running in the cluster. 
+    kubectl-strace starts a priviledged Pod and attaches an strace instance
+    to one or many running Pods, displaying or collecting the results for later.
+    Read more documentation see: https://github.com/michaelwasher/kstrace
```
Comment (ahmetb) [inline plugins/strace.yaml:14]: a bit of grammatical issue here but ok, you can fix later.

File: plugins/strace.yaml (hunk 4 above)
Comment (ahmetb) [reply plugins/strace.yaml:7]: We don't have any enforcement of genericclioptions (it's a recommendation). Same with client-go/pkg/client/auth –but if you don't import this, you're severely limiting your user base.

**Comment by ahmetb** [comment]
/lgtm
/approve

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**